FluentTerminal.App.ViewModels/TerminalViewModel.cs

```diff
@@ -341,11 +341,22 @@ private async void OnKeyboardCommandReceived(object sender, string command)
                 var selection = await _terminalView.GetSelection().ConfigureAwait(true);
                 _clipboardService.SetText(selection);
             }
-            else if (command == nameof(Command.Paste))
+            else if ((command == nameof(Command.Paste)) || (command == nameof(Command.PasteWithoutNewlines)))
             {
-                var content = await _clipboardService.GetText().ConfigureAwait(true);
+                string content = await _clipboardService.GetText().ConfigureAwait(true);
                 if (content != null)
                 {
+                    // Use the shell profile line-ending translation method to translate the line endings of the pasted buffer
+                    // or delete all of the instances of the newline pattern by replacing with the empty string.
```

Comment:
Maybe inline the comments inside the if-statement? To be honest I don't think they're really even necessary...

Reply:
Resolved by converting that section to a `switch() { }`.